src/Microsoft.Azure.WebJobs.Host/Blobs/BlobPathSource.cs

```diff
@@ -18,28 +18,27 @@ public static IBlobPathSource Create(string pattern)
                 throw new FormatException("Blob paths must not be null.");
             }
 
-            string containerNamePattern;
-            string blobNamePattern;
             int slashIndex = pattern.IndexOf('/');
             bool hasBlobName = slashIndex != -1;
 
+            string containerNamePattern = pattern;
+            string blobNamePattern = String.Empty;
             if (hasBlobName)
             {
-                // There must be at least one character before the slash and one character after the slash.
-                bool hasNonEmptyBlobAndContainerNames = slashIndex > 0 && slashIndex < pattern.Length - 1;
-
-                if (!hasNonEmptyBlobAndContainerNames)
-                {
-                    throw new FormatException("Blob paths must be in the format container/blob.");
-                }
-
                 containerNamePattern = pattern.Substring(0, slashIndex);
                 blobNamePattern = pattern.Substring(slashIndex + 1);
             }
-            else
+
+            // There must be at least one character before the slash and one character after the slash.
+            bool hasNonEmptyBlobAndContainerNames = slashIndex > 0 && slashIndex < pattern.Length - 1;
+
+            if ((hasBlobName && !hasNonEmptyBlobAndContainerNames) || containerNamePattern.Contains('\\'))
```

Comment:
Input/output bindings use `BlobPath` - you can see where I standardized the error above. They already handled a backslash.